### DISCUSSION TITLE
请问你们在nas里面是如何更新的

### DISCUSSION THREAD

**Discussion by kyon777** [AlistGo/alist#4954] [General]
alist安装在nas的 docker系统里面 ，装载路径/opt/alist/data，我每次更新都要先把alist的配置导出来在重新拉取最新版本的镜像，创建新的容器再导入配置， 大佬们有更简单的方法吗？ 例如像linux那样 直接再usr目录更新下 alist那样。

**Comment by li-peifeng** [comment]
把数据目录映射出来就行了，没必要每次导来导去
